test/jasmine/tests/hover_label_test.js

```diff
@@ -670,8 +670,8 @@ describe('hover on fill', function() {
                 var transformParts = hoverText.attr('transform').split('(');
                 expect(transformParts[0]).toEqual('translate');
                 var transformCoords = transformParts[1].split(')')[0].split(',');
-                expect(+transformCoords[0]).toBeCloseTo(labelPos[0], 0, labelText + ':x');
-                expect(+transformCoords[1]).toBeCloseTo(labelPos[1], 0, labelText + ':y');
+                expect(+transformCoords[0]).toBeCloseTo(labelPos[0], -1, labelText + ':x');
+                expect(+transformCoords[1]).toBeCloseTo(labelPos[1], -1, labelText + ':y');
```

Comment:
This particular one I was not able to get completely consistent across all the test runners, maybe it has something to do with differences in `getBoundingClientRect` between them? Anyway, there's a consistent ~1-2px discrepancy that's captured happily by this looser test.